utils/dwarf.c

```diff
@@ -326,6 +328,8 @@ static int setup_dwarf_info(const char *filename, struct uftrace_dbg_info *dinfo
 
 	dinfo->file_type = elf_file_type(dinfo);
 
+	close(fd); // Close the file descriptor to release the acquired resource
```

Comment:
I'm not sure if it's ok to close the fd here.  libdw/libdwfl will access the file later and they assume the file is open during the processing.

Reply:
Yes, it should not be closed here